tests/phpunit/CRM/Dedupe/DedupeFinderTest.php

```diff
@@ -518,7 +516,7 @@ public function testDupesByParams() {
    * Locks in expected params
    *
    */
-  public function hook_civicrm_findDuplicates($dedupeParams, &$dedupeResults, $contextParams) {
+  public function hook_civicrm_findDuplicates($dedupeParams, $dedupeResults, $contextParams) {
```

Comment:
This doesn't match the dev docs, and the return at the bottom. Are the docs wrong?

Otherwise everything in the PR looks good.

Reply:
Actually I think the docs are right and the test is half-right. The signature was correct, but the return is wrong, it just doesn't come up because the hook implementation never changes the value.

Reply:
@demeritcowboy yeah - I normally remove the pass by ref in extension hooks too if they don't leverage it - just makes it all clearer

Reply:
@demeritcowboy I didn't want this tidy up to be blocking on the actual fix - https://github.com/civicrm/civicrm-core/pull/25527 - so I reverted it back & added php doc block to ignore it.